autopilot/autopilot.go

```diff
@@ -698,3 +729,166 @@ func (ap *Autopilot) hostsHandlerPOST(jc jape.Context) {
 	}
 	jc.Encode(hosts)
 }
+
+func countUsableHosts(cfg api.AutopilotConfig, cs api.ConsensusState, fee types.Currency, currentPeriod uint64, rs api.RedundancySettings, gs api.GougingSettings, hosts []hostdb.Host) (usables uint64) {
+	gc := worker.NewGougingChecker(gs, cs, fee, currentPeriod, cfg.Contracts.RenewWindow)
+	for _, host := range hosts {
+		usable, _ := isUsableHost(cfg, rs, gc, host, 0, 0)
```

Comment:
Passing `0,0` here is not ideal. Maybe we should split this up or skip `hostScore` if `minScore` is 0 in `isUsableHost`